lib/SIL/IR/SILDeclRef.cpp

```diff
@@ -387,6 +387,12 @@ bool SILDeclRef::hasUserWrittenCode() const {
   llvm_unreachable("Unhandled case in switch!");
 }
 
+bool SILDeclRef::isGetter() const {
+  if (auto *accessor = dyn_cast_or_null<AccessorDecl>(getFuncDecl()))
+    return accessor->isGetter();
```

Comment:
Is this also going to be true for getters of ClangImporter-synthesized properties? We probably don't want to emit all of those.

Reply:
@slavapestov are those ObjC properties that are imported into Swift? Couldn't they also be accessed in the debugger?

Reply:
When evaluating an objective-C method call, the compiler generates the right thunks with shared linkage. That's why we don't want to force emission of those symbols if they're not needed. So perhaps the mayDelay computation needs to check the SIL linkage.